addon/components/frost-select.js

```diff
@@ -145,8 +149,8 @@ export default Component.extend(PropTypeMixin, {
    */
   invalidFilter (data, displayItems) {
     return (
-      _.isArray(data) &&
-      _.isArray(displayItems) &&
+      Array.isArray(data) &&
```

Comment:
Should we be using destructing in this case as well?

`const { isArray } = Array`